ISLE/isleapp.cpp

```diff
@@ -457,6 +484,63 @@ SDL_AppResult SDL_AppEvent(void* appstate, SDL_Event* event)
 		}
 		break;
 	}
+	case SDL_EVENT_GAMEPAD_BUTTON_DOWN: {
+		{
+			if (event->gbutton.button == SDL_GAMEPAD_BUTTON_SOUTH) {
```

Comment:
On 3DS this maps to the B button (I just tested). I'm not sure what the best way of ifdefing or setting a value on 3DS to map to the A button would be. I guess B button is fine, people could easily figure it out and get used to it, but, /shrug

Reply:
Use https://wiki.libsdl.org/SDL3/SDL_GamepadType to detect if it should be one or the other.